23/02/2021 01:06:12 PM : INFO - Data path checked..
23/02/2021 01:06:12 PM : INFO - Model save path checked..
23/02/2021 01:06:12 PM : INFO - config JSON path checked..
23/02/2021 01:06:12 PM : INFO - Tensorboard Visualization path checked..

xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Running training with the following parameters: 
23/02/2021 01:06:12 PM : INFO - Cleaning Visualization path of older tensorboard files...


data_path : ./dataset
model_path : ./model_checkpoints
vis_path : ./vis_checkpoints
model_save_name : meme.pt
no_model_checkpoints : False
remove_checkpoints : False
config : config/uniter-base.json
feature_path : ./dataset/own_features
pretrained_model_file : uniter-base.pt
max_txt_len : 60
max_bb : 100
min_bb : 10
num_bb : 36
optimizer : adam
loss_func : bce_logits
optimize_for : aucroc
scheduler : warmup_cosine
beta1 : 0.9
beta2 : 0.999
batch_size : 16
num_workers : 0
gradient_accumulation : 2
max_grad_norm : 5
pos_wt : 1.0
lr : 3e-05
warmup_steps : 500
weight_decay : 0.001
max_epoch : 1
lr_decay_step : 3
lr_decay_factor : 0.8
patience : 5.0
early_stop_thresh : 0.001
seed : 43
log_every : 2000
fc_dim : 64
dropout : 0.2
filter_text : False
no_normalize_img : True
train_filename : train.jsonl
upsample_multiplier : 0
note : 
race_gender_hidden_size : 8
device : cuda
n_classes : 1

xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
filter text False
Loaded dataset contains  8500 samples
Traceback (most recent call last):
  File "train_uniter.py", line 637, in <module>
    upsample_multiplier=config["upsample_multiplier"])
  File "/home/astro/Documents/UvA/Block 4 - NLP2/Multimodal NLP/Multimodal-NLP/data/meme_dataset.py", line 64, in __init__
    self._prepare_data_list()
  File "/home/astro/Documents/UvA/Block 4 - NLP2/Multimodal NLP/Multimodal-NLP/data/meme_dataset.py", line 204, in _prepare_data_list
    self._load_gender_race_probs()
  File "/home/astro/Documents/UvA/Block 4 - NLP2/Multimodal NLP/Multimodal-NLP/data/meme_dataset.py", line 249, in _load_gender_race_probs
    self.data.gender_race_probs[i] = gender_race_probs[image_id]
IndexError: list assignment index out of range
